db/migrate/20190930121136_add_player_id_to_admin_users_and_contact.rb

```diff
@@ -0,0 +1,10 @@
+class AddPlayerIdToAdminUsersAndContact < ActiveRecord::Migration[5.2]
+  def change
+    add_column :admin_users, :player_id, :string
+
+    create_table :contacts do |t|
+      t.references :user, null: false
+      t.text       :text, null: false
```

Comment:
This should be split into two files.
One migration for creating the table and the other for adding the column